### PR TITLE
Update learningcontent-get.md and Api Overview.md

### PR DESCRIPTION
**Pull request by SritejaKampara**
Updating the Right URL for Learning Content GET Api

Co-Authored-By: roymalabika <108811446+roymalabika@users.noreply.github.com>
Co-Authored-By: hemashar <108335902+hemashar@users.noreply.github.com>

### CODE REVIEW

**Review by SteveWilkins1123 — DISMISSED** [review]

**Review by SteveWilkins1123 — COMMENTED** [review]
## Allowlisting <What is this a permission?>
As part of Registration, we are going to Allow list your AppId for the tenant. <what do you mean by we are going to allow? Not now; in the future? If so then you cannot mention anything about a feature that will be available in the future.

To enable content ingestion from Partner for the tenant. You should provide your App Id https://aka.ms/contentsync 
<What is the reference to aka.ms.contentsync.> 


Do you want to say
To enable ingestion of content by partners from a tenant, as part of Registration, listing AppId for a tenant will be enabled by default.

File: api-reference/beta/api/learningcontent-get.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ One of the following permissions is required to call this API. To learn more, in
 }
 -->
 ``` http
-GET /employeeExperience/learningProviders/{learningProviderId}/learningContents/{externalId}
+GET /employeeExperience/learningProviders/{learningProviderId}/learningContents(externalId='{externalId}')
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/learningcontent-get.md:33]: Line 33:
Can one of the authors please confirm if there was a schema change to make externalId a parameter in the query?

Or is learningContents still a nav property of learningProvider? If that's the case, can you confirm if the following should work without explicitly specifying the property name externalId, and where {externalId} is a value for the externalId property? (Like you noticed, Graph Explorer doesn’t support your API right now.)

GET /employeeExperience/learningProviders/{learningProviderId}/learningContents(‘{externalId}’)

In general, if there’s a schema change that entailed an API review, please include the internal roadmap item # at the top of the Conversation tab in GitHub for your PR. More info in step #6 in the [workflow](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211773/Content-workflow?anchor=workflow-for-internal-contributors-to-add-new-content-or-update-content) section.

File: api-reference/beta/api/learningcontent-get.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ One of the following permissions is required to call this API. To learn more, in
 }
 -->
 ``` http
-GET /employeeExperience/learningProviders/{learningProviderId}/learningContents/{externalId}
+GET /employeeExperience/learningProviders/{learningProviderId}/learningContents(externalId='{externalId}') 
 ```
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/learningcontent-get.md:35]: This comment applies to all the example queries for the learningProvider and learningContent API topics:

Unlike the "HTTP request" section, example request should always specify the fully qualified URL including the domain name. For example, the GET request for learningContent should be:

`GET https://graph.microsoft.com/beta/employeeExperience/learningProviders/13727311-e7bb-470d-8b20-6a23d9030d70`

Can you please update all the example queries in your API set?

BTW, can you confirm if Graph Studio generated these example requests without fully qualified query URLs?

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
I entered a couple of comments when your request came in and one of us started responding. 

In general, it would be good if your PR can follow our [content workflow](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211773/Content-workflow?anchor=workflow-for-internal-contributors-to-add-new-content-or-update-content) so that a reviewer can dedicate their attention while reviewing your PR. Otherwise there can be oversights that affect the customer experience for your API, and general expectations that take concerted efforts across the docs to build.

I myself have a few PRs and other tasks on hand, so I'll likely let someone else follow up on this PR and complete the review as it moves up the review queue.

File: api-reference/beta/api/learningcontent-get.md (hunk 1 above)
Comment (angelgolfer-ms) [reply api-reference/beta/api/learningcontent-get.md:33]: On the other hand, if learningContents is now redefined as a function, then we should document the function parameter in a different way. See more info in [Parameters](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211760/Parameters), and an example for the [reminderView](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211760/Parameters) function.

File: api-reference/beta/resources/viva-learning-api-overview.md [hunk 3]
```diff
@@ -41,6 +41,9 @@ Use the employee learning APIs to do the following:
 - [Get the specified metadata of a provider's ingested content](../api/learningcontent-get.md) in Viva Learning.
 - [Delete the specfied metadata of a provider's ingested content](../api/learningprovider-delete-learningcontents.md).
 
+## Allowlisting
```
Comment (SteveWilkins1123) [inline api-reference/beta/resources/viva-learning-api-overview.md:44]: What is AllowListing? Is it a permission?

File: api-reference/beta/resources/viva-learning-api-overview.md [hunk 4]
```diff
@@ -41,6 +41,9 @@ Use the employee learning APIs to do the following:
...
+As part of Registration, we are going to Allow list your AppId for the tenant. To enable content ingestion from Partner for the tenant. You should provide your App Id https://aka.ms/contentsync
```
Comment (SteveWilkins1123) [inline api-reference/beta/resources/viva-learning-api-overview.md:45]: Going to allow implies that this will be done at some point in the future. We should not talk about features that are coming soon. 

I would rephrase as
To enable partners to ingest content from a tenant, as part of Registration, listing the AppId for a tenant will be enabled by default.  

I am not sure what "You should provide your App Id https://aka.ms/contentsync" this has to do with the rest of the information

File: api-reference/beta/resources/viva-learning-api-overview.md (hunk 3 above)
Comment (roymalabika) [reply api-reference/beta/resources/viva-learning-api-overview.md:44]: Hi Steve , We are currently allowing specific Application id to connect to our Graph API. The form is provided to enter the Application id by the Provider

Comment (SteveWilkins1123) [reply api-reference/beta/resources/viva-learning-api-overview.md:44]: So you may want to say something like this:
To enable the ingestion of partner content from a tenant, as part of Registration, providers can call this API to register their Application IDs with the tenant.

So if a provider registers an AppID with a tenant they are allowed to ingest content from the tenant even if it is owned by someone else? Or does the owner of a tenant call this API to register a provider APPID?

File: api-reference/beta/resources/viva-learning-api-overview.md (hunk 4 above)
Comment (isvargasmsft) [inline api-reference/beta/resources/viva-learning-api-overview.md:45]: Lines 44 and 45: Following Steve's comments, Could you please expand this description? We can chat on the side for efficiency, I'd like to understand better what is Allowlisting and how it works from both the developer and IT admin perspective.

File: api-reference/beta/api/learningcontent-get.md (hunk 1 above)
Comment (isvargasmsft) [inline api-reference/beta/api/learningcontent-get.md:33]: Could you please clarify if this change also applies to the List operation? If not, could you please clarify the difference? If it does, please update the List topic as well.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @SritejaKampara, 

Thank you very much for your work in this PR. 

Please take a look at Angela and Steve's comments as well as a couple of additional comments I left. I am also reaching out to you on the side to chat a bit more about the Allowlisting feature. 

Thank you!

Isaac Vargas

**Comment by SritejaKampara** [comment]
This PR is no more applicable as allowlisting is removed now, and also the change to learningcontent-get.md is included in https://github.com/microsoftgraph/microsoft-graph-docs/pull/17997/commits
